style.css

```diff
@@ -925,3 +925,173 @@ h3 {
   position: relative;
   right: -90%;
 }
+
+.btProjMob {
+  width: 50%;
+  height: 10%;
+  background: #36b37e;
+  border-style: none;
+  border-radius: 4px;
+}
+
+.TextButtonProjects {
+  color: #fff;
+  width: auto;
+  height: auto;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 24px;
+  pointer-events: none;
+  justify-content: center;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+/* Style for MobilePopUp */
+
+.MobilePopUp {
+  position: sticky;
+  position: -webkit-sticky;
+  top: 0;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: auto;
+  background-image: url(img/Overlay.svg);
+  background-position: center;
```

Comment:
- [ ] In the mobile version please allow the overlay to scroll so that the visitor can see the buttons at the bottom of the modal :recycle: